samples/NuGet.config

```diff
@@ -7,6 +7,47 @@
         <!--
         -->
     </packageSources>
+    <!-- 
+        https://docs.microsoft.com/en-us/nuget/consume-packages/package-source-mapping
+        Define mappings by adding package patterns beneath the target source. 
+
+        packages restored from "Local Output":
+            Xamarin.AndroidX.*
+            Xamarin.Kotlin.*
+            Xamarin.KotlinX.*
+            Xamarin.Google.AutoValue.Annotations
+            GoogleGson
+            Xamarin.Google.Guava
+            Xamarin.Google.Guava.* 
+            Xamarin.Google.J2Objc.Annotations
+        packages restored from nuget.org:
+            everything else 
+    -->
+    <packageSourceMapping>
+        <!-- key value for <packageSource> should match key values from <packageSources> element -->
+        <packageSource key="Local Output">
+            <package pattern="Xamarin.AndroidX.*" />
+            <package pattern="Xamarin.Google.Android.Material" />
+            <package pattern="Xamarin.Kotlin.*" />
+            <package pattern="Xamarin.KotlinX.*" />
+            <package pattern="Xamarin.Google.AutoValue.Annotations" />
+            <package pattern="GoogleGson" />
+            <package pattern="Xamarin.Google.Guava" />
+            <package pattern="Xamarin.Google.Guava.*" />
+            <package pattern="Xamarin.AndroidX.*" />
+            <package pattern="Xamarin.Google.J2Objc.Annotations" />
+        </packageSource>
+        <packageSource key="nuget.org">
+            <package pattern="*" />
+            <package pattern="Xamarin.AndroidX.*" />
```

Comment:
I.m investigating that.

For example you cannot exclude something. We have Crypto.Security 1.0.0.7 and MAUI sample chokes, because it has strict/fixed dependency on 1.1.0-alpha03 and you cannot say take 1.0.0.7 from local source and for MAUI 1.1.0-alpha03.